go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.0
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489
+	golang.org/dl v0.0.0-20210315215437-3c153da08ce0 // indirect
```

Comment:
This seems unrelated to the current PR 🤔

Reply:
Hmm, yeah. I think that must have snuck in by running `make serve-book`. So it may be needed but just omitted before. I will remove it, run that make target again, and see if it gets pulled back in.

Reply:
Nope, it didn't. Not sure where that came from, but I will drop it from this patch.